File: isAlpha/isalpha.py
'''The `isalpha()` method is a Python string method that checks if all the characters in a given string are 
alphabetic characters (letters) or not. It returns `True` if all characters are letters, and `False` otherwise.

//Here's an example:
'''
string1 = "Hello"
string2 = "123"

print(string1.isalpha())  # Output: True, as all characters are letters
print(string2.isalpha())  # Output: False, as there are non-letter characters

'''
In the context of the pangram-checking code, `ch.isalpha()` is used to check if each character in the string is a letter before adding it to the set of unique letters. 
This ensures that non-alphabetic characters are not considered when checking for a pangram.
'''

File: isAlpha/isalpha.cpp
/*In C++, the `isalpha()` function is part of the `<cctype>` header, and it is used to check if a character 
is an alphabetic character (a letter).

Here's an example usage within a C++ program:

*/
#include <iostream>
#include <cctype>
using namespace std;
int main() {
    char ch1 = 'A';
    char ch2 = '1';

    if (isalpha(ch1)) {
        cout << ch1 << " is an alphabetic character." << std::endl;
    } else {
        cout << ch1 << " is not an alphabetic character." << std::endl;
    }

    if (isalpha(ch2)) {
        cout << ch2 << " is an alphabetic character." << std::endl;
    } else {
        cout << ch2 << " is not an alphabetic character." << std::endl;
    }

    return 0;
}
/*
In this example, `isalpha(ch)` returns a non-zero value if `ch` is an alphabetic character, and 0 otherwise. 
The output of the program would be:


A is an alphabetic character.
1 is not an alphabetic character.
*/
